staging/src/k8s.io/legacy-cloud-providers/azure/azure_managedDiskController.go

```diff
@@ -229,6 +229,10 @@ func (c *ManagedDiskController) DeleteManagedDisk(diskURI string) error {
 
 	disk, rerr := c.common.cloud.DisksClient.Get(ctx, resourceGroup, diskName)
 	if rerr != nil {
+		if rerr.HTTPStatusCode == http.StatusNotFound {
+			klog.V(2).Infof("azureDisk - disk(%s) is already deleted", diskURI)
+			return nil
+		}
```

Comment:
Is it possible to add unit test for this case? This would prevent any potential regressions.

Reply:
it's a little difficult since this requires a mock for DisksClient.Get, and also this stagin repo is in maintenance mode, only accpects bug fixes, so that's it.